anime-list-master.xml

```diff
@@ -8742,7 +8742,7 @@
   <anime anidbid="2599" tvdbid="85066" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Gin'yuu Mokushiroku Meine Liebe</name>
   </anime>
-  <anime anidbid="2600" tvdbid="136251" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="2600" tvdbid="136251" defaulttvdbseason="1" episodeoffset="" tmdbid="763471" imdbid="">
```

Comment:
Not the same thing
https://www.themoviedb.org/movie/763471

Reply:
Corrected, if there is a (correct) value in tvdbid. Should I remove leave the tmdbid=""imdbid="" blank?

https://www.themoviedb.org/movie/663680

Reply:
I don't think I follow, but blank ID is preferable over wrong ID

Reply:
I updated the tvdbid to the correction one, but I left the tmdbid. In the README.md if there is a show it can only have an TVDBID. From look at this line in the readme.

"tmdbid/imdbid - themoviedb.org/imdb.com id. Only used for one-off titles like movies. Do not use tmdbid for series!"

So I think I need to remove it and clean up any other entery with those values filled and a tvdbid filled?